text/0000-process-exit.md

```diff
@@ -0,0 +1,90 @@
+- Feature Name: exit
+- Start Date: 2015-03-24
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+
+Add a function to the `std::process` module to exit the process immediately with
+a specified exit code.
+
+# Motivation
+
+Currently there is no stable method to exit a program in Rust with a nonzero
+exit code without panicking. The current unstable method for doing so is by
+using the `exit_status` feature with the `std::env::set_exit_status` function.
+
+This function has not been stabilized as it diverges from the system APIs (there
+is no equivalent) and it represents an odd piece of global state for a Rust
+program to have. One example of odd behavior that may arise is that if a library
+calls `env::set_exit_status`, then the process is not guaranteed to exit with
+that status (e.g. Rust was called from C).
+
+The purpose of this RFC is to provide at least one method on the path to
+stabilization which will provide a method to exit a process with a nonzero exit
```

Comment:
s/a nonzero/an arbitrary/, or, perhaps, there’s any reason why calling `exit(0)` is invalid?